### PR TITLE
chore(docs): Adds security recommendations for Azure ARM templates to forward logs

### PR DESCRIPTION
**Pull request by jsubirat**
* What problems does this PR solve?
Adds a new section to explain how to strengthen the security of a log forwarding setup deployed using any of our Azure ARM templates.

* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
This documentation explains how to manually perform the steps to reconfigure an already existing log forwarding setup to use the enhancements that were introduces in [this PR](https://github.com/newrelic/newrelic-azure-functions/pull/79) and [this PR](https://github.com/newrelic/newrelic-azure-functions/pull/80).

### CODE REVIEW

**Comment by jsubirat** [comment]
Hi folks, this PR is still being worked on, I'll move the Draft flag once we're ready for a review from your side. Many thanks in advance!

**Comment by jsubirat** [comment]
Hi docs team, the PR is now ready to be reviewed, thanks!

File: src/content/docs/logs/forward-logs/azure-log-forwarding.mdx [hunk 1]
```diff
@@ -167,6 +177,118 @@ SELECT * FROM Log where plugin.type='azure'
 
 If no data appears after you enable our log management capabilities, follow our [standard log troubleshooting procedures](/docs/logs/log-management/troubleshooting/no-log-data-appears-ui/).
 
+## Security recommendations for your Azure resources [#azure-security-recommendations]
+
+In May 2013, we included some security enhancements in our ARM templates that modify the settings in the Storage Account and the Function App deployed using those templates. Azure resources are secured by default and these changed settings prevent this secure default state from being modified.
```
Comment (rkbodenner) [inline src/content/docs/logs/forward-logs/azure-log-forwarding.mdx:182]:
```suggestion
In May 2023, we included some security enhancements in our ARM templates that modify the settings in the Storage Account and the Function App deployed using those templates. Azure resources are secured by default and these changed settings prevent this secure default state from being modified.
```

**Review by rkbodenner — CHANGES_REQUESTED** [review]
This looks ready to ship, with the one date typo to be fixed.

**Review by x8a — APPROVED** [review]
Thanks a lot for your edits!

**Pull request merged**